odc_k8s/helm_operator.tf

```diff
@@ -0,0 +1,60 @@
+variable "helm_operator_version" {
```

Comment:
please rename this file flux_helm_operator

Reply:
done